lib/order.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'menu'
+
+class Order
+
+attr_reader :order_list, :menu
+
+  def initialize(menu)
+    @menu = menu
+    @order_list = {}
+  end
+
+  def add(dish, quantity)
+    @order_list[dish] = quantity
```

Comment:
What happens if you want to add something more than once? You want pasta but then matey over here wants pasta too and adds another one, it looks like this will only add one lot to the order, rather than two